lib/ecto/changeset.ex

```diff
@@ -528,6 +528,11 @@ defmodule Ecto.Changeset do
         {changes, errors, valid?}
       :invalid ->
         {changes, [{key, {"is invalid", [type: type, validation: :cast]}} | errors], false}
+      {:invalid, custom_errors} ->
+        {msg, custom_errors} = Keyword.pop(custom_errors, :message)
+        msg = msg || "is invalid"
```

Comment:
Should we raise if the message is not present? Thoughts @wojtekmach?

Reply:
I would make it optional to reduce boilerplate. 

For example, `{:error, reason: :invalid_country}` would need to rewritten to `{:error, reason: :invalid_country, message: "is invalid"}` even if the user does not want to provide a custom error message.